pkgs/development/compilers/llvm/13/default.nix

```diff
@@ -247,11 +247,18 @@ let
                else stdenv;
     };
 
-    libcxxabi = callPackage ./libcxxabi {
-      inherit llvm_meta;
-      stdenv = if stdenv.hostPlatform.useLLVM or false
+    libcxxabi = let
+      stdenv_ = if stdenv.hostPlatform.useLLVM or false
```

Comment:
I think what you want to do here is test `stdenv.cc.isClang`. `useLLVM` isn't true for Darwin, for example. (Unless that changed recently?)

Reply:
I'm not exactly sure what the intention behind that conditional expression is, so I left it unchanged in this PR. Does your suggestion apply to other parts of the code too? I see the same conditional expression used everywhere throughout the LLVM packaging code.

Reply:
Hmm, I guess leave it if you've tested it on Darwin anyway?